components/data-components/contact-form.tsx

```diff
@@ -0,0 +1,210 @@
+import { HomePageType } from '@/lib/interface'
+import { useState } from 'react'
+import { Formik, Form, Field, ErrorMessage } from 'formik'
+
+import { Button } from '../data-components/button'
+// import { ContactFormField } from '../data-components/contact-form-field'
+
+export const ContactForm = ({ data }: HomePageType) => {
+  console.log('ContactForm => data', data)
+  const [formFieldValues] = useState({
+    firstname: '',
+    lastname: '',
+    email: '',
+    phone: '',
+    message: '',
+  })
+
+  const [isInputValid, setIsInputValid] = useState(false)
+  const [isSubmitMsg, setIsSubmitMsg] = useState(false)
+
+  {
+    /* Dynamically render a component if time permits */
```

Comment:
Remove the commented code we don't need it

Reply:
done